sample_xblocks/filethumbs/filethumbs.py

```diff
@@ -59,9 +64,9 @@ def student_view(self, context=None):  # pylint: disable=W0613
         frag = Fragment(unicode(html_str))
 
         if not self.fs.exists("thumbsvotes.json"):
-            with self.fs.open("thumbsvotes.json", "wb") as f:
-                json.dump({'up': 0, 'down': 0}, f)
-                f.close()
+            with self.fs.open('thumbsvotes.json', 'wb') as file_output:
+                json.dump({'up': 0, 'down': 0}, file_output)
+                file_output.close()
```

Comment:
Throwing in my chips with @stvstnfrd here - better var names keep on the path to self documenting code.